test/chunking-form/samples/manual-chunk-is-entry-match/_expected/es/main.js

```diff
@@ -1,4 +1,4 @@
-console.log('dep');
+import './generated-dep.js';
```

Comment:
```ts
	options: {
		input: {
			main: 'main.js'
		},
		output: {
			manualChunks: {
				main: ['main']
			}
		}
	}
```
Setting an entry as manualChunk will now prevent Rollup from merging deps.js into main.js. The developer will have to make sure not to pass a same file name to both `input` and `manualChunk`.

Reply:
From the intention of the test, I would suggest to manually add "dep" to the "main" chunk